runtime/compiler/env/J9ClassEnv.hpp

```diff
@@ -147,6 +162,7 @@ class OMR_EXTENSIBLE ClassEnv : public OMR::ClassEnvConnector
    bool isClassFinal(TR::Compilation *comp, TR_OpaqueClassBlock *);
    bool hasFinalizer(TR::Compilation *comp, TR_OpaqueClassBlock *classPointer);
    bool isClassInitialized(TR::Compilation *comp, TR_OpaqueClassBlock *);
+   bool isClassVisible(TR::Compilation *comp, TR_OpaqueClassBlock *sourceClass, TR_OpaqueClassBlock *destClass);
```

Comment:
Could you add Doxygen comments for this method?

Reply:
Addressed in 307194d